### PR TITLE
BASH-22 Get tests running

### PR DESCRIPTION
**Pull request by csduarte**
**Description**
This PR updates spectron to 3.7.2 to make sure that all tests run successfully.

- [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
 - [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
 - [x] executed `npm run lint:js` for proper code formatting

### CODE REVIEW

**Comment by yuya-oc** [comment]
I think Electron and Spectron should use same minor version. So I will merge when upgrading Electron.
https://github.com/electron/spectron/blob/master/README.md

If my understanding is not correct, please let us know.

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "mocha": "^3.3.0",
     "mocha-circleci-reporter": "0.0.2",
     "npm-run-all": "^4.0.2",
-    "spectron": "~3.6.2",
+    "spectron": "^3.7.2",
```
Comment (yuya-oc) [inline package.json]: Use `~` for spectron.

**Comment by jasonblais** [comment]
@yuya-oc Thoughts about upgrading our dependancies, including to Electron v1.7.6? Looks like that's the second stable release of Electron v1.7.

I'm not sure how big of an impact the major version update will be.

**Comment by yuya-oc** [comment]
@jasonblais Recently I have finished reviewing Electron changelogs from 1.6.x to 1.7.6. I will create a PR to upgrade.

**Comment by jasonblais** [comment]
@csduarte Closing this PR in favor of https://github.com/mattermost/desktop/pull/596, which upgrades both Electron and Spectron dependancies.

**Pull request closed without merging**